numpy_cpp-custom-matrix/example.cpp

```diff
@@ -38,9 +38,7 @@ Matrix<double> Multiply(const Matrix<double> &a, const Matrix<double> &b) {
   size_t b_cols = b.shape()[1];
   if (a_cols != b_rows)
     throw std::length_error("Matrix 'a' and 'b' are inconsistent");
-  std::vector<size_t> shape(2);
-  shape[0] = a_rows;
-  shape[1] = b_cols;
+  std::vector<size_t> shape = {a_rows, b_cols};
```

Comment:
'shape' is the name of a function in matrix.h. Here, use another name (for example 'c') for the variable.